packages/pds/src/api/com/atproto/server/getAccountInviteCodes.ts

```diff
@@ -62,9 +69,10 @@ export default function (server: Server, ctx: AppContext) {
             const forUser = await dbTxn.db
               .selectFrom('invite_code')
               .where('forUser', '=', requester)
+              .where('createdBy', '!=', 'admin') // dont count admin-gifted codes aginast the user
               .selectAll()
               .execute()
-            if (forUser.length > userCodes.length + toCreate) {
+            if (forUser.length > routineCodes.length + toCreate) {
```

Comment:
When I read through this again with the additional "routine" idea in the mix, the `forUser` name tripped me up a little bit.  If it were something like `finalRoutineCodes` it might be a little easier to pick-up what's happening here.

Reply:
a good call, I dig 👍